jenkins.sh

```diff
@@ -42,7 +39,8 @@ virtualenv $WORKSPACE/.venv
 PYTHON=$WORKSPACE/.venv/bin/python
 PIP=$WORKSPACE/.venv/bin/pip
 (cd synapse ; $PYTHON synapse/python_dependencies.py | xargs -n1 $PIP install)
-
+$PIP install psycopg2
+$PIP install lxml
```

Comment:
> This is copied from the synapse jenkins.sh

Comment this then please so if the Synapse `jenkins.sh` changes and we don't need to do that anymore, we know we can safely nuke this bit as well.